src/app/Upload.svelte

```diff
@@ -99,9 +126,26 @@
   on:drop|preventDefault={handleDrop}
   on:dragover|preventDefault={handleDragOver}
   on:dragleave|preventDefault={handleDragLeave}>
-  <img role="presentation" src="icons/upload-solid.svg" alt="" />
+  <img role="presentation" src="icons/upload-solid.svg" alt=""
+/>
   <span>Drag and drop here</span>
   <span>or</span>
   <label for={UPLOAD_INPUT_ID} class="upload-btn">Browse file</label>
-  <input id={UPLOAD_INPUT_ID} type="file" on:change={handleChange} />
+  <input id={UPLOAD_INPUT_ID} type="file" on:change={handleChange} data-cy="upload-file" />
 </div>
+
+<!--
+  TODO: remove this once https://github.com/eddieantonio/predictive-text-studio/pull/58 is merged.
+-->
+{#if files.size > 0}
+<button data-cy="download-kmp" on:click={workInProgressCreateModel}>
```

Comment:
**[Question]**: if this is not a throw away code/testing piece, a test-id would allow easy testing

Reply:
This is throw-away code!

But for the test-id, we're using `data-cy`, since that is [the official recommendation from Cypress](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).

Reply:
Hmmm, for the reason of maintainability and scaleability, `test-id` still seems to be better in the long run. For instance, if we scrape cypress off the project in the future, we would have to spend time in refactoring the name

Reply:
We're using `data-cy` in this project :)